scripts/docker-run.sh

```
@@ -0,0 +1,104 @@
+
+echo -e "${BLUE}Running Docker Compose for $NETWORK...${NC}"
+echo -e "${YELLOW}Compose file: $COMPOSE_FILE${NC}"
+echo -e "${YELLOW}Command: $COMMAND${NC}\n"
+
+# Run docker compose
+docker compose -f "$DOCKER_DIR/$COMPOSE_FILE" "${COMMAND[@]}"
+
+echo -e "\n${GREEN}Command completed successfully!${NC}"
+
+# Show helpful tips based on command
+if [[ "$COMMAND" == "up"* ]]; then
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)